### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by okineadev** [lishaduck/lishaduck#64] [Q&A] [answered]
What is your name?

**Comment by lishaduck** [comment, marked as answer]
It's Eli.

**Reply by okineadev** [reply]
> It's Eli.

Is this a male name?

**Reply by lishaduck** [reply]
Yes.

**Reply by okineadev** [reply]
![Screenshot_2024-11-04-21-40-39-856_com android chrome](https://github.com/user-attachments/assets/e8749062-289b-473b-ac0e-155c7ab5ae94)

**Reply by lishaduck** [reply]
Correct.

**Reply by okineadev** [reply]
> Correct.

Are you jewish 🤩?

**Reply by lishaduck** [reply]
Nope.

**Reply by lishaduck** [reply]
<img width="579" alt="Screenshot 2024-11-04 at 5 12 20 PM" src="https://github.com/user-attachments/assets/28d9d6cd-8e5f-4d56-8226-221bf7b806b1">

Are you?

**Reply by okineadev** [reply]
> <img width="579" alt="Screenshot 2024-11-04 at 5 12 20 PM" src="https://github.com/user-attachments/assets/28d9d6cd-8e5f-4d56-8226-221bf7b806b1">
> 
> Are you?

Watta sigma 🤯

This is the first time I see this 😅

I'm not Jewish

**Reply by okineadev** [reply]
How old are you?

**Reply by lishaduck** [reply]
I'd rather not say.
If you're that curious, look at my orgs. That'd give you a vague idea, methinks.
